frontend/src/app/shared/components/editor/components/ckeditor/op-ckeditor.component.ts

```diff
@@ -274,6 +276,11 @@ export class OpCkeditorComponent extends UntilDestroyedMixin implements OnInit,
           this.saveRequested.emit();
           evt.stop();
         }
+
+        if (data.keyCode === KeyCodes.ESCAPE) {
+          debugLog('Sending escape request from CKEditor.');
```

Comment:
debugLog :eyes:

Reply:
😅 I followed the saveRequested convention [just above](https://github.com/opf/openproject/blob/dev/frontend/src/app/shared/components/editor/components/ckeditor/op-ckeditor.component.ts#L273) - no strong opinion on retaining it but it should be harmless as it's only logged in dev mode either way